civicpy/exports/civic_vcf_writer.py

```diff
@@ -0,0 +1,100 @@
+from civicpy import civic
+import datetime
+from civicpy.__version__ import __version__
+import vcfpy
+
+
+class CivicVcfWriter():
+    """
+    :param path filepath: A file path for the VCF output file
+    :param [civic.exports.CivicVcfRecord] vcf_records: A list of :class:`civic.exports.CivicVcfRecord` objects to write to the output file
+    :param string version: VCF version
+    """
+
+    CSQ_DESCRIPTION = 'Consequence annotations from CIViC. Format: {}'.format('|'.join([
+        'Allele',
+        'Consequence',
+        'SYMBOL',
+        'Entrez Gene ID',
+        'Feature_type',
+        'Feature',
+        'HGVSc',
+        'HGVSp',
+        'CIViC Variant Name',
+        'CIViC Variant ID',
+        'CIViC Variant Aliases',
+        'CIViC Variant URL',
+        'CIViC Molecular Profile Name',
+        'CIViC Molecular Profile ID',
+        'CIViC Molecular Profile Aliases',
+        'CIViC Molecular Profile URL',
+        'CIViC HGVS',
+        'Allele Registry ID',
+        'ClinVar IDs',
+        'CIViC Molecular Profile Score',
+        'CIViC Entity Type',
+        'CIViC Entity ID',
+        'CIViC Entity URL',
+        'CIViC Entity Source',
+        'CIViC Entity Variant Origin',
+        'CIViC Entity Status',
+        'CIViC Entity Significance',
+        'CIViC Entity Direction',
+        'CIViC Entity Disease',
+        'CIViC Entity Therapies',
+        'CIViC Entity Therapy Interaction Type',
+        'CIViC Evidence Phenotypes',
+        'CIViC Evidence Level',
+        'CIViC Evidence Rating',
+        'CIViC Assertion ACMG Codes',
+        'CIViC Assertion AMP Category',
+        'CIViC Assertion NCCN Guideline',
+        'CIVIC Assertion Regulatory Approval',
+        'CIVIC Assertion FDA Companion Test',
```

Comment:
Not sure if this was intentional or a typo

```suggestion
        'CIViC Assertion Regulatory Approval',
        'CIViC Assertion FDA Companion Test',
```